awx/api/serializers.py

```diff
@@ -4931,7 +4931,8 @@ class Meta:
             'node_state': {'initial': Instance.States.INSTALLED, 'default': Instance.States.INSTALLED},
             'hostname': {
                 'validators': [
-                    MaxLengthValidator(limit_value=255),
```

Comment:
why change from 255 to 250 max len here?

Reply:
@AlanCoding because if you look at the definition of the model field, you'll see it is 250, not 255.  These need to match.